### ISSUE TITLE
F.logsigmoid not accepting tensor

### ISSUE THREAD

**Issue by ruotianluo** [pytorch/pytorch#962]
I don't know how torch.nn.functional should work, but apparently F.sigmoid can accept tensor but F.logsigmoid can't.

**Comment by fmassa** [comment]
I think for the moment none of the functions in `nn.functional` are supposed to accept tensors, and only Variables. Automatically wrapping Tensors in functions is planned though, so it will be supported sometime in the future.

**Comment by apaszke** [comment]
As @fmassa said, we don't accept tensors in `nn.functional`.
